components/test_raftstore/src/server.rs

```diff
@@ -335,6 +335,12 @@ impl ServerCluster {
             None
         };
 
+        if cfg.tikv.storage.api_version == 2 {
```

Comment:
What's the magic number `2`?

Reply:
I shouldn't have written this, it has been edited.like `ApiVersion::V2 == F::TAG`